# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: toolkit/components/passwordmgr/test/mochitest/mochitest.ini
# Commit: bfe21f274ed5
# Full Hash: bfe21f274ed5a4630a3503bcb875fed04f90533c
# Author: Anny Gakhokidze <agakhokidze@mozilla.com>
# Date: 2020-04-16 03:00:04
# Regressor Bug: 1594529
# File Overlap Count: 1
# Description:
#   Bug 1594529 - Create LoadInfo for subdocuments directly in parent process with DocumentChannel. r=mattwoodrow,nika
#   
#   Currently, with Fission enabled we are not able to create a proper LoadInfo
#   object when doing a subdocument load because we do not have access to a loading
#   context if the load is happening inside of an OOP frame. To solve this problem,
# ==============================================================================

diff -r 29ff609063c2 -r bfe21f274ed5 toolkit/components/passwordmgr/test/mochitest/mochitest.ini
--- a/toolkit/components/passwordmgr/test/mochitest/mochitest.ini	Wed Apr 15 18:53:22 2020 +0000
+++ b/toolkit/components/passwordmgr/test/mochitest/mochitest.ini	Wed Apr 15 18:53:06 2020 +0000
@@ -151,7 +151,6 @@
 [test_prompt.html]
 skip-if = os == "linux" || toolkit == 'android' # Tests desktop prompts
 [test_prompt_async.html]
-fail-if = fission # bug 1582117
 skip-if = toolkit == 'android' || (debug || asan || tsan) # Tests desktop prompts # Bug 1603659
 support-files = subtst_prompt_async.html
 [test_prompt_http.html]
